commit 4b022ed32bb3d729fd21d2e33483fda03c6a6333
Author: Jiří Klimeš <jklimes@redhat.com>
Date:   Tue Nov 23 19:06:58 2010 +0100

    core: correct TIMESTAMP type; fixes potential crash when time() is 32bit
    
    The Crash appeared on some platforms in glib while processing variable
    arguments.

diff --git a/src/system-settings/nm-default-wired-connection.c b/src/system-settings/nm-default-wired-connection.c
index 1cda3d9..0d19dea 100644
--- a/src/system-settings/nm-default-wired-connection.c
+++ b/src/system-settings/nm-default-wired-connection.c
@@ -163,7 +163,7 @@ constructor (GType type,
 		      NM_SETTING_CONNECTION_AUTOCONNECT, TRUE,
 		      NM_SETTING_CONNECTION_UUID, uuid,
 		      NM_SETTING_CONNECTION_READ_ONLY, priv->read_only,
-		      NM_SETTING_CONNECTION_TIMESTAMP, time (NULL),
+		      NM_SETTING_CONNECTION_TIMESTAMP, (guint64) time (NULL),
 		      NULL);
 
 	g_free (id);
